src/main/resources/project/procedures/preamble.groovy

```diff
@@ -234,6 +243,29 @@ public class ElectricCommander {
             
     }
 
```

Comment:
Lets not create the zone here. We'll require the user to select a value from existing zones.